forms-flow-web/src/components/Form/constants/constants.scss

```diff
@@ -0,0 +1,15 @@
+.client-table{
+    min-height: 400px;
+}
+.sort-icon{
```

Comment:
is this file is necessary? because we have a  list.scss  file so all tables will come under the list.js fileright